frontend/src/components/messstelle/MessquerschnittInfo.vue

```diff
@@ -40,7 +40,7 @@
                         >
                         <v-list-item-subtitle
                             >Lage Messquerschnitt:
-                            {{ messstelle.standort }}</v-list-item-subtitle
+                            {{ messquerschnitt.standort }}</v-list-item-subtitle
```

Comment:
Hier soll doch die Lage angezeigt werden und nicht der Standort. Die Lage steht in dem Feld Beschreibung